templates/ts/app/entry.worker.ts

```diff
@@ -154,7 +154,7 @@ const handlePush = (event: PushEvent) => {
     silent: data.silent ? data.silent : false,
   };
 
-  self.registration.showNotification(title, {
+  return self.registration.showNotification(title, {
```

Comment:
:+1:

Reply:
`ServiceWorkerRegistration.showNotification()` returns a promise that resolves to undefined, so the return value is redundant anyway